src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -32,11 +33,37 @@ class Riddle {
 	end?: number
 	active: boolean = false
 	rushing: boolean = false
+	gcdsInEachFist: { [fistId: number]: number } = {
+		[FISTLESS]: 0,
+		[STATUSES.FISTS_OF_EARTH.id]: 0,
+		[STATUSES.FISTS_OF_FIRE.id]: 0,
+		[STATUSES.FISTS_OF_WIND.id]: 0,
+	}
+	expectedGcds: number = 11 // Baseline GL4 RoFs are 11 GCDs
 
 	constructor(start: number) {
 		this.start = start
 		this.casts = []
 	}
+
+	get gcds() {
+		return this.casts.filter(event => {
+			const action = getDataBy(ACTIONS, 'id', event.ability.guid) as TODO
+			return action && action.onGcd
+		}).length
+	}
+
+	get elixirField() {
```

Comment:
ubernit;
```suggestion
	get elixirFields() {
```

---

i really feel that this repetitive accessor pattern is getting a bit out of hand, but there's prior art so it's ok for now. gonna need to figure out a way to clean them up